### PR TITLE
Fix/prepare env

### PR DESCRIPTION
**Pull request by 3asm**

### CODE REVIEW

**Review by najibraihan — APPROVED** [review]

File: .github/workflows/pytest-linux.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        python-version: ${{ matrix.python-version }}
+    - name: Init and update data submodule.
+      run: |
+        git submodule init
+        git submodule update
+    - name: Install dependencies.
+      run: |
+        python -m pip install --upgrade pip
+        pip install .[testing]
+        pip install .[agent]
```
Comment (deadly-panda) [inline .github/workflows/pytest-linux.yml]: These 2 lines are not needed in this project.
Apply this and the next comment in the other workflow files too please.

File: .github/workflows/pytest-linux.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Init and update data submodule.
+      run: |
+        git submodule init
+        git submodule update
```
Comment (deadly-panda) [inline .github/workflows/pytest-linux.yml]: Do we need this?

**Review by deadly-panda — DISMISSED** [review]

File: .github/workflows/pytest-linux.yml (hunk 2 above)
Comment (3asm) [reply .github/workflows/pytest-linux.yml]: Good catch.

**Comment by deadly-panda** [comment]
I do not see the submodule init&update removals?

**Pull request merged**